frontend/src/components/common/ItemSortMethod.tsx

```diff
@@ -18,6 +19,22 @@ type ItemSortMethodProps = {
 
 const ItemSortMethod = ({ onChange }: ItemSortMethodProps) => {
   const { formatMessage } = useIntl();
+  const themeContext = useContext(ThemeContext);
+
+  const customStyles = {
+    menu: (styles: any) => ({
+      ...styles,
+      background: `${themeContext.theme === LIGHT ? "#ffffff" : "var(--dark)"}`,
+    }),
+    option: (styles: any) => ({
+      ...styles,
+      background: `${themeContext.theme === LIGHT ? null : "var(--dark)"}`,
```

Comment:
It's possible to do it the same way here, instead of check the theme with a condition, it is better to go on ```src/styles/index.scss```, inside ```:root``` add a variable that takes ```none``` as value and inside ```:root.dark``` add a variable the same variable with a dark color as value.

Same remarks for lines 33 and 34

Reply:
thank for your suggestion, i have add your suggestion, view result below

light mode

![Capture d’écran de 2021-07-28 19-12-29](https://user-images.githubusercontent.com/67915678/127378422-25d8ae2f-b60f-4ad1-af8a-03e73f711663.png)


dark mode

![Capture d’écran de 2021-07-28 19-12-46](https://user-images.githubusercontent.com/67915678/127378492-748855ec-e147-4687-8717-874622c16eb6.png)